src/sandesh/coding_exam_3/RemoveVowelsFromString.java

```diff
@@ -0,0 +1,40 @@
+/*WAP to return a string by removing all the vowels from the given String.
+ Output should be printed in main method.
+
+ input :  globant india test automation engineer
+
+ output :  glbnt nd tst tmtn ngnr*/
+
+package sandesh.coding_exam_3;
+
+public class RemoveVowelsFromString {
+	
+	String getStringWithoutVowel(String eachSubString) {
+		String tp = eachSubString;
+		for(int index = 0; index < eachSubString.length(); index++)
```

Comment:
if you are using reg ex then not require for loop

Reply:
sure Harsh, trying as you mentioned. Thank you for your inputs.